README.md

```diff
@@ -1,54 +1,210 @@
+```sh
+composer create-project symfony/skeleton <project-name>
+```
+or use an existing one (symfony >= 3.4)
 
-Open composer.json and add to the attribute require this line
+Navigate to the project's directory and run
+```sh
+composer config extra.symfony.allow-contrib true
+```
+This will allow symfony to use the recipes-contrib (via Symfony Flex) that will automate
+most of the configuration.
 
+To install the commercetools symfony-bundle open composer.json and add to the attribute 
+`require` the package `"commercetools/symfony-bundle"`
+and run `composer install` or directly run the following on the command line
 ```sh
 composer require commercetools/symfony-bundle
 ```
 
-Add the Commercetools Bundle to your application kernel
+Open `.env` or create a `.env.local` [1] file on root directory and edit the following 
+lines to add  your credentials. The credentials can be retrieved through 
+`Commercetools Merchant Center > Settings > Developer Settings`, while you create a new 
+API Client. Note that for security reasons you cannot retrieve the `CTP_CLIENT_SECRET` 
+for clients created in the past.
+
+```dotenv
+CTP_CLIENT_ID=<your client id>
+CTP_CLIENT_SECRET=<your client secret>
+CTP_PROJECT_KEY=<your project id>
+CTP_AUTH_URL=https://auth.commercetools.com or https://auth.commercetools.co
+CTP_API_URL=https://api.commercetools.com or https://api.commercetools.co
+CTP_SCOPES=<your desired scopes>
+```
+
+[1] To read more about the differences between the usage of .env and .env.local you can
+continue [here](https://symfony.com/doc/current/configuration/dot-env-changes.html) and/or
+[here](https://symfony.com/blog/new-in-symfony-4-2-define-env-vars-per-environment)
+
+### Verify everything works
+
+To verify that your configuration works, after adding your client credentials on the .env file
+as explained above, you may run on the command line:
+`bin/console commercetools:project-info`. If everything is set up correctly, this should return
+the details regarding your project like:
+
+```
+Project's key: super-cool-project-4
+Project's name: my eshop at commercetools
+Countries: DE, US
+Currencies: EUR
+Languages: en
+Created at: 2019-02-04T11:28:49+00:00
+Messages: disabled
+```
+
+if so, it means that your project can now communicate with commercetools platform via the client
+you just configured.
+
+## Usage
+
+commercetools symfony bundle consists of 8 smaller bundles (The dependencies mentioned are
+only the additional ones, required for bundle-specific functionalities)
+
+- CartBundle
+- CatalogBundle
+- CtpBundle
+- CustomerBundle
+    - depends: `symfony/security-bundle`
+- ReviewBundle
+- SetupBundle
+    - depends: `symfony/console`
+- ShoppingListBundle
+- StateBundle
+    - depends: `symfony/console`, `symfony/workflow`, `twig/extensions`
+    
+
+By default, the enabled bundles are `CtpBundle` & `CustomerBundle` for all environments, and
+additionally `SetupBundle` & `StateBundle` only for dev environment. To overview, 
```

Comment:
```suggestion
`SetupBundle` & `StateBundle` are only enabled for for development environments. To overview, 
```

Also – "to overview?" 🤔

Reply:
I meant to write something like that on this file, you can see which bundles are enabled, and that you are able disable them, or you may enable other ones too if you want.

Reply:
Ahh. Then I'd literally say:

"To see which bundles are enabled..."